.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,108 @@
+name: "🐛 Bug Report"
+description: "If something isn't working as expected 🤔."
+title: "[Bug]: "
+labels: ["i: needs triage"]
```

Comment:
I removed the `i: bug` label here, since at least half of the "bug reports" we get aren't actual bugs.